app.json

```diff
@@ -32,6 +32,10 @@
      }
    ],
    "env": {
+     "SITE_URL": {
+       "description": "Access url to the instance e.g. https://<app name>.herokuapp.com without a trailing slash.",
```

Comment:
I would be more strict here. Instead of **e.g.**, this **has to be** "https://<app name>.herokuapp.com", right?

Reply:
:+1: